bundles/org.openhab.binding.pushbullet/src/main/java/org/openhab/binding/pushbullet/internal/handler/PushbulletHandler.java

```diff
@@ -86,139 +88,270 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
     @Override
     public void initialize() {
-        logger.debug("Start initializing!");
-        config = getConfigAs(PushbulletConfiguration.class);
+        logger.debug("Starting {}", thing.getUID());
+
+        PushbulletConfiguration config = getConfigAs(PushbulletConfiguration.class);
+
+        if (config.getToken().isEmpty()) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "Undefined token.");
+            return;
+        }
 
-        // Name and Token are both "required", so set the Thing immediately ONLINE.
-        updateStatus(ThingStatus.ONLINE);
+        httpClient.setConfiguration(config);
 
-        logger.debug("Finished initializing!");
+        scheduler.execute(() -> retrieveAccountInfo());
+
+        updateStatus(ThingStatus.UNKNOWN, ThingStatusDetail.CONFIGURATION_PENDING);
```

Comment:
For reference: https://www.openhab.org/docs/concepts/things.html#status-details
```suggestion
        updateStatus(ThingStatus.UNKNOWN);
```